src/client/containers/PullRequestSummary/ReviewSection/ReviewerList/index.js

```diff
@@ -29,25 +25,104 @@ const getStateClass = (state: string) => {
 
 const getClass = (icon: string, state: string) => `fa ${icon} ${getStateClass(state)}`
 
-const ReviewerList = (props: ReviewerListType) => {
-  if (props.reviewers.length === 0) {
-    return null
-  }
-  const tooltipOverlay = <Tooltip id="tooltip">{props.tooltip}</Tooltip>
-  return (
-    <li>
-      <span>
-        <OverlayTrigger placement="left" overlay={tooltipOverlay}>
-          <i
-            style={{ marginRight: '12px', width: '12px', marginBottom: '6px' }}
-            className={getClass(props.icon, props.approval)}
-          />
-        </OverlayTrigger>
-        {props.reviewers.map(r => r.user.fullName || r.user.username).join(', ')}
-      </span>
-    </li>
+export const getReviews = createSelector(
```

Comment:
General selectors should definitely go in ducks/; control-specific ones could either go there or locally. I don't feel too strongly either way. Let's discuss when Kate's back.